src/auxinfo/participant.rs

```diff
@@ -225,24 +228,29 @@ impl AuxInfoParticipant {
             message.id(),
             &self.other_participant_ids.clone(),
         )?;
-        let mut messages = vec![];
 
         if r1_done {
```

Comment:
(I know you are just refactoring the code so this isn't against you). I hate this code. So weird that we call `handle_round_two_msg` from within `handle_round_one_msg` and return the results in a variable called "round_one_outcome" 😷😷😷